### ISSUE TITLE
multiple Discriminator networks

### ISSUE THREAD

**Issue by loliverhennigh** [soumith/ganhacks#8]
Thanks for the list! Multiple discriminator networks have helped me out a fair amount. Would this make your list as a [notsure] method? I haven't tried using RL stuff where you keep checkpoints of past Gs and Ds so maybe that is a similar but much better method.

**Comment by soumith** [comment]
yes, for sure. They have appeared in multiple papers in various forms. Please send a PR, and I'll merge it in.

**Comment by zhangqianhui** [comment]
@loliverhennigh Can you shared same detailed about multiplt discriminator networks?

**Comment by zhangqianhui** [comment]
before I test using multiple discriminator networks, but the result is not better.
